File: GoServer/Model/review.go
package Model

type Review struct {
	Id        			string      	`json:"_id" bson:"_id"`
	ProductId        	string      	`json:"productId" bson:"productId"`
	CustomerId        	string      	`json:"customerId" bson:"customerId"`
	Rating 				int 			`json:"rating" bson:"rating"`
	Headline 			string			`json:"headline" bson:"headline"`
	Description 		string			`json:"description" bson:"description"`
	Votes 				int			`json:"votes" bson:"votes"`
	ReviewImages 		[]string		`json:"reviewImages" bson:"reviewImages"`
}

File: src/app/Models/review.ts
export class Review {
    _id: string;
    customerId: string; 
    productId: string; 
    rating: number; 
    headline: string;
    description: string; 
    votes: number; 
    reviewImages: string[];

    constructor(
        _id: string, 
        customerId: string, 
        productId: string, 
        rating: number, 
        headline: string,
        description: string, 
        votes: number, 
        reviewImages: string[],
    ) {
        this._id = _id;
        this.customerId = customerId;
        this.productId = productId;
        this.rating = rating;
        this.headline = headline;
        this.description = description;
        this.votes = votes;
        this.reviewImages = reviewImages;
    }

}